docs/v2/ios.md

```diff
@@ -1,53 +1,149 @@
-# iOS Installation
+# React native Agora iOS Installation
+#### Minimum compatibility
+* React native 0.58.x
+* IOS SDK 8.0+
+* Android 5.0+ x86 arm64 armv7a
 
-# Prerequisites
-* You should installed and linked `react-native-agora`
-* Xcode 10.0+
-* cocoapods
+#### 1. open up your terminal and write this :
+When it comes to the version nr you can put any version you like as long as it it compatibility
+```sh
+$ react-native init agoraguide --version react-native@0.58
+```
 
-```bash
-# Use npm
-npm install react-native-agora
+#### 2. cd into your project folder
+```sh
+$ cd agoraguide
 ```
-or
+#### 3. install npm package react-native-agora
+
+with npm
 
-```bash
-# Use yarn
-yarn add react-native-agora
+
+```sh
+$ npm install --save react-native-agora
 ```
-then you need to link the library
 
-```bash
-# Link the library
-react-native link react-native-agora
+or yarn
+
+```sh
+$ yarn add react-native-agora
 ```
 
 
-```bash
-# installed ruby first then run it
-gem install cocoapods
+#### 5. cd into your ios folder
+```sh
+$ cd ios
 ```
 
-# Step 1. install pod dependencies
-```bash
-cd ios
-pod install
+#### 6. we need to initialize pod
+if you don't have cocopods installed run this command in your terminal
+```sh
+$ sudo gem install Cocoapods
```

Comment:
I don't think you probably need the sudo here. U should grant the current user gem install permission.

Reply:
Totally agree about that, let me change that

Reply:
Removed the `sudo` command